### DISCUSSION TITLE
Price includes Greenpower?

### DISCUSSION THREAD

**Discussion by jbanyer** [amberelectric/public-api#41] [Q&A] [answered]
Hi Myles,

Can you confirm that the total price `perKwh` includes Greenpower, for those on Greenpower plans? Looking at some prices for me it appears that it does include Greenpower.

**Comment by madpilot** [comment, marked as answer]
It does

**Reply by madpilot** [reply]
I should be clear: it includes it if you have it enabled in your account.

If that is a error, ping info@amber.com.au and we can fix it up
